packages/ibm-products/src/components/Tearsheet/_tearsheet.scss

```diff
@@ -270,7 +272,7 @@ $motion-duration: $duration-moderate-02;
   .#{$block-class}__main {
     display: flex;
     flex-direction: row;
-    background-color: $layer;
+    background-color: $background;
```

Comment:
We need to overwrite this for narrow tearsheets as they use `layer` as their background instead of `background`:

```scss
.#{$block_class}--narrow .#{$block_class}__main {
  background-color: $layer;
}
```

Reply:
Updated as requested